client/signup/steps/about/index.jsx

```diff
@@ -515,7 +514,7 @@ class AboutStep extends Component {
 											'e.g. Fashion, travel, design, plumber, electrician'
 										) }
 										onChange={ this.onSiteTopicChange }
-										suggestions={ Object.values( hints ) }
+										suggestions={ this.siteTopicSuggestions }
```

Comment:
Should we add the `value=` attribute here too, like we do in the [site topic step](https://github.com/Automattic/wp-calypso/pull/29024/files#diff-39d9db3436b90753194497ec9f54092cR94)? Presently, the About step does not _remember_ the site topic input if I press the Back button to come to this step. 

If we plan to add the `value=` attribute here, then we should not send the English value in [setSurvey()](https://github.com/Automattic/wp-calypso/pull/29024/files#diff-7949db3a74b97e79a5ff08e5b794a6d8R209) so that the translated value is shown if I press the Back button (explained in more detail below in my comment on the site topic step).